frontend/src/components/SessionTile.js

```diff
@@ -4,16 +4,16 @@ export default function SessionTile({ session }) {
     return (
         <Wrapper>
             <span>
-                <img src={session.thumbnail} alt="" />
+                <img src={session.imageUrl} alt="" />
             </span>
             <div>
-                <span>{session.startDate}</span>
+                <span>{session.startDateTimestamp.substring(0, 11)}</span>
```

Comment:
checkout JavaScripts `Date` object and the various ways to format dates. Use a `<time>` tag here.